gno.land/cmd/gnoland/config_init.go

```diff
@@ -0,0 +1,66 @@
+package main
+
+import (
+	"context"
+	"errors"
+	"flag"
+	"fmt"
+
+	"github.com/gnolang/gno/tm2/pkg/bft/config"
+	"github.com/gnolang/gno/tm2/pkg/commands"
+)
+
+var errInvalidConfigOutputPath = errors.New("invalid config output path provided")
+
+type configInitCfg struct {
+	outputPath string
+}
+
+// newConfigInitCmd creates the new config init command
+func newConfigInitCmd(io commands.IO) *commands.Command {
+	cfg := &configInitCfg{}
+
+	cmd := commands.NewCommand(
+		commands.Metadata{
+			Name:       "init",
+			ShortUsage: "config init [flags]",
+			ShortHelp:  "Initializes the Gno node configuration",
+			LongHelp: "Initializes the Gno node configuration locally with default values, which includes" +
+				" the base and module configurations",
+		},
+		cfg,
+		func(_ context.Context, _ []string) error {
+			return execConfigInit(cfg, io)
+		},
+	)
+
+	return cmd
+}
+
+func (c *configInitCfg) RegisterFlags(fs *flag.FlagSet) {
+	fs.StringVar(
+		&c.outputPath,
+		"output-path",
+		"./config.toml",
```

Comment:
I'm actually glad you brought this up -- I held off standardizing the default paths (both for `genesis` in the genesis PR and `config`) until I implement `gnoland init`, since the default value here should dictate both for `genesis` and `config`. But yes, you're completely right, they will be in sync

Reply:
okay, sounds good